packages/gensx/src/checkpoint.ts

```diff
@@ -594,6 +594,11 @@ export class CheckpointManager implements CheckpointWriter {
       // Handle root node case
       if (!this.root) {
         this.root = node;
+
+        // If the workflow name is set, update the root node name.
+        if (this.workflowName) {
+          this.root.componentName = this.workflowName;
```

Comment:
@EvanBoyle just FYI. I realized we weren't doing anything with the name parameter, so for now I am just setting the `componentName` on the root node, as this is what we are currently using in the console to extract the name of the workflow. Eventually we need to update the checkpoint API to separate the two concepts.